recipes/ambertools/meta.yaml

```diff
@@ -26,30 +25,36 @@ requirements:
     - patch
     - tcsh
     - bison
+    - bzip2
+    - zlib
+    - openmp
   host:
-    - python <3
+    - python >3
     - pip
-    - zlib
-    - bzip2
+    - numpy
+    - scipy
+    - matplotlib
+    - boost
   run:
-    - python <3
-
-# bc csh flex gfortran g++ xorg-dev zlib1g-dev libbz2-dev patch
+    - python >3
+    - numpy
+    - scipy
+    - matplotlib
+    - apbs
 
 test:
   commands:
     - antechamber -h
+    - teLeap -h
     - tleap -h
-    - prepgen -h
+    - parmchk2 -h
```

Comment:
if you are sure that the python packages are working you could try this test as well:

```
  imports:
    - pytraj
```

Reply:
It doesn't work. But I don't think we need this.